File: c_py_solution/0404_py/middle.py
def solution(a1, a2):
    res = []
    for outer in range(len(a1)):
        temp = []
        for inner in range(len(a1[outer])):
            temp.append(a1[outer][inner] + a2[outer][inner])
        res.append(temp)
    print(res)
    return res


a1 = [[1, 2], [2, 3]]
a2 = [[3, 4], [5, 6]]
solution(a1, a2)

File: b_js_solution/0404_js/middle.js
function solution(a1, a2) {
    return a1.map((row, i) =>
        // val = a1 (1,2,2,3)
        row.map((val, j) => val + a2[i][j])
    )
}

const a1 = [[1,2],[2,3]];
const a2 = [[3,4],[5,6]];
console.log(solution(a1, a2))